php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/customizer/CustomizerPhpUnit.java

```diff
@@ -603,4 +632,23 @@ private void processUpdate() {
         }
     }
 
+    private static class PhpUnitVersionComboBoxModel extends DefaultComboBoxModel<PhpUnitVersion> {
+
+        private static final long serialVersionUID = 5850175934190021687L;
+
+        public PhpUnitVersionComboBoxModel() {
+            this(null);
+        }
+
+        public PhpUnitVersionComboBoxModel(PhpUnitVersion preselected) {
+            super(PhpUnitVersion.values());
+
+            if (preselected != null) {
+                setSelectedItem(preselected);
```

Comment:
how about
setSelectedItem(preselected != null ? preselected : PhpUnitVersion.getDefault());

Reply:
Not needed, no? Or am I missing something?

Reply:
it is a syntactic proposal , one line of code instead of five, feel free to ignore if you don't like it.

Reply:
I am just checking whether we are preventing some kind of error here, like NPE or similarly.

Reply:
Or, I may remove this if I fix this issue without selecting the version.